content/articles/domains-uk.markdown

```diff
@@ -65,6 +65,10 @@ To transfer a UK domain (i.e. example.uk or example.co.uk) into DNSimple, follow
 
 To transfer a UK domain out of DNSimple, [contact support](https://dnsimple.com/contact) and provide your domain name and your new registrar's IPS tag.
 
+## Updating the registrant name
+
+It is possible to change most values in the Whois record for a UK domain, but the registrant name must be changed through Nominet's Online Services (Nominet is the Registry for UK domains). It is possible to [register for Nominet Online Services](https://secure.nominet.org.uk/auth/login.html) with the email address you have in your registrant information and update the value for the registrant name with them.
```

Comment:
@weppos I made a change in 8e21c9d. Let me know if you want it to change, but I'm going to ship this for now.